master/static/srv/proto_get_trial_workloads_with_batch_metrics.sql

```diff
@@ -0,0 +1,72 @@
+WITH validations_vt AS (
```

Comment:
TODO: try to make this conditional in the same query as `proto_get_trial_workloads`